app/queries/proxies_for_owner_and_watcher_query.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ProxiesForOwnerAndWatcherQuery
```

Comment:
Not really convinced about this being a query object.

https://github.com/3scale/porta/pull/2230#discussion_r528623780

Reply:
it's more Query object than a Service for me. i would keep it this way.